sdk/attestation/azure-security-attestation/README.md

```diff
@@ -0,0 +1,153 @@
+
+
+## Getting started
+### Adding the package to your project
+Maven dependency for the Azure Attestation  library. Add it to your project's POM file.
+
+[//]: # ({x-version-update-start;com.azure:azure-security-attestation;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-security-attestation</artifactId>
+    <version>1.0.0-beta.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+### Prerequisites
+- A [Java Development Kit (JDK)][jdk_link], version 8 or later.
+- [Azure Subscription][azure_subscription]
+- An existing [Azure Attestation][azure_attestation]. If you need to create an attestation instance, you can use the [Azure Cloud Shell][azure_cloud_shell] to create one with this Azure CLI command. Replace `<your-resource-group-name>` and `<your-instance-name>` with your own, unique names:
+
+    ```bash
+    az attestation create --resource-group <your-resource-group-name> --name <your-key-vault-name>
+    ```
+
+### Authenticate the client
+In order to interact with the Azure Attestation service, your client must present an Azure Active Directory bearer token to the service.
+
+The simplest way of providing a bearer token is to use the  `DefaultAzureCredential` authentication method by providing client secret credentials is being used in this getting started section but you can find more ways to authenticate with [azure-identity][azure_identity].
+
+
+## Key concepts
+
+The Microsoft Azure Attestation service runs in two separate modes: "Isolated" and "AAD". When the service is running in "Isolated" mode, the customer needs to 
+provide additional information beyond their authentication credentials to verify that they are authorized to modify the state of an attestation instance.
+
+There are four major components in the preview  
+
+The *Key concepts* section should describe the functionality of the main classes. Point out the most important and useful classes in the package (with links to their reference pages) and explain how those classes work together. Feel free to use bulleted lists, tables, code blocks, or even diagrams for clarity.
+
+## Examples
+
+Include code snippets and short descriptions for each task you listed in the [Introduction](#azure-attestation-client-library-for-java) (the bulleted list). Briefly explain each operation, but include enough clarity to explain complex or otherwise tricky operations.
+
+If possible, use the same example snippets that your in-code documentation uses. For example, use the snippets in your `examples.py` that Sphinx ingests via its [literalinclude](https://www.sphinx-doc.org/en/1.5/markup/code.html?highlight=code%20examples#includes) directive. The `examples.py` file containing the snippets should reside alongside your package's code, and should be tested in an automated fashion.
```

Comment:
examples.py?

Reply:
Apparently the readme.md in the sdk\template\readme.md file is actually for Python, not Java, thus there's a BUNCH of python references which shouldn't be there...